docs/migration.rst

```diff
@@ -97,7 +97,7 @@ The following example shows and explains how to create commands effortlessly and
     @interactions.option(4, name="opt2", description="This is an option.")
     @interactions.option(interactions.Channel, name="opt3", required=True)
     async def command_with_options(
-        ctx, opt1: str, opt2, int, opt3: interactions.Channel = None
+        ctx, opt3: interactions.Channel, opt1: str = None, opt2: int = None,
```

Comment:
This is a bit confusing - I feel like these options should be renamed in a more logical way if you want to use this format.

Reply:
So make opt1 required instead?

Reply:
Could also just rename the channel to `opt1` and the string to `opt3`.

Reply:
That isn't what the decorators say tho

Reply:
Hmm I think I messed up the decorators
Think it would make more sense if the first two were required and the third was not
Could you please change these?